test/unit/adb-commands-specs.js

```diff
@@ -254,7 +254,15 @@ describe('adb commands', () => {
         await adb.setWifiAndData({wifi:true});
         mocks.adb.verify();
       });
-      it('should call shell with correct args when only data', async () => {
+      it('should call shell with correct args when turning only wifi off', async () => {
+        mocks.adb.expects("shell")
+          .once().withExactArgs(['am', 'start', '-n', 'io.appium.settings/.Settings',
+                                  '-e', 'wifi', 'off'])
+          .returns("");
+        await adb.setWifiAndData({wifi:false});
```

Comment:
I realize you're just following the surrounding code here, which is great, but all of these should be `{wifi: <value>}` (i.e., with a space after the colon). mind cleaning it all up while you're in here?